build_library/vm_image_util.sh

```diff
@@ -594,8 +594,42 @@ label coreos
   kernel /coreos/vmlinuz
   append initrd=/coreos/cpio.gz coreos.autologin
 EOF
-    mkisofs -v -l -r -J -o $2 -b isolinux/isolinux.bin -c isolinux/boot.cat -no-emul-boot -boot-load-size 4 -boot-info-table .
-    isohybrid $2
+
+    # create fat image of grub for EFI
+    mformat -i isolinux/efi.img -C -v EFI -f 2880
+    mmd -i isolinux/efi.img EFI
+    mmd -i isolinux/efi.img EFI/BOOT
+
+    mcopy -i isolinux/efi.img -o "${base_dir}"/boot/EFI/boot/* "::EFI/BOOT/"
+
+    # syslinux won't work on arm64, and there is no point in isohybrid
+    # for arm64/uefi.
```

Comment:
Why is there no point to isohybrid? Isn't that needed to be able to use the ISO as a usb stick/drive/thingy instead of a cdrom? Or does isohybrid just not work for arm64?